src/components/ButtonPanel.js

```diff
@@ -35,4 +38,12 @@ const ButtonPanel = () => (
   </div>
 );
 
+ButtonPanel.propTypes = {
+  onClick: PropTypes.func,
+};
+
+ButtonPanel.defaultProps = {
+  onClick: e => e.target,
```

Comment:
There should be an handleclick function ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png)
calling e.target on ButtonPanel is not ideal, buttonPanel is a react component not a DOM element.

Reply:
Thanks for your review. I'll change those details ASAP

Reply:
Done!